.github/workflows/lint.yml

```diff
@@ -54,10 +54,6 @@ jobs:
           key: npm-lint-cache
       - name: lint the selenium javascript tests
         run: |
-          cd diagrams
+          cd docs/diagrams
           npm install
           npm run lint
-      - name: test building diagram
-        run: |
-          cd diagrams
-          node make_overview.js
```

Comment:
Yes, in that we lint the Javascript code in the docs/diagram directory, but no in the sense that there is nowhere else here that we're "testing" running the diagrams builder.  So it was a choice to remove it here, as I believe creates noise as its not actually linting, and otherwise an adds limited value to what we're otherwise testing in total in our CI. 

The longer answer: The Diagrams code which generates two static files for documentation is code we are likely to only rely upon very infrequently and it's not couple with any of our other code, so nothing can break it but changes to the diagrams code itself, which only happens if we're generating diagrams and then presumably it works for the job. I don't think we need to be running it everything CI run.